packages/cubejs-athena-driver/src/AthenaDriver.ts

```diff
@@ -63,6 +63,24 @@ export class AthenaDriver extends BaseDriver implements DriverInterface {
     this.athena = new Athena(this.config);
   }
 
+  public async tableColumnTypes(table: string) {
+    const [schema, name] = table.split('.');
+
+    const columns = (await this.query(
+      `SELECT columns.column_name as ${this.quoteIdentifier('column_name')},
+             columns.table_name as ${this.quoteIdentifier('table_name')},
+             columns.table_schema as ${this.quoteIdentifier('table_schema')},
+             columns.data_type  as ${this.quoteIdentifier('data_type')}
+      FROM information_schema.columns
+      WHERE table_name = ${this.param(0)} AND table_schema = ${this.param(1)}
+      ORDER BY 1, 2, 3, 4`,
```

Comment:
Nit: `ORDER BY 1` suffices, the other columns are redundant.

Reply:
You are right, agree